lib/fathead/caniuse/redirects.txt

```diff
@@ -0,0 +1,217 @@
+::selection element, ::selection css pseudo element
+selection element, ::selection css pseudo element
+selection pseudo elements, ::selection css pseudo element
+css selection element, ::selection css pseudo element
+ambient lights, ambient light
+ambient lights api, ambient light api
+ambient-lights, ambient-light
```

Comment:
No, they have the same content, But there's an entry for each in the `output.txt` file, and also in the `article.txt` file.